### DISCUSSION TITLE
missing GO data for Aspergillus nidulans

### DISCUSSION THREAD

**Discussion by murphyte** [geneontology/helpdesk#395] [Q&A]
Hi GO -- GO data for Aspergillus nidulans FGSC A4 (taxid 227321) seems to have gone missing. Is that expected? It's no longer listed on either of these pages:
http://current.geneontology.org/products/pages/downloads.html
http://current.geneontology.org/annotations/

Thanks for checking.
-Terence

**Comment by kltm** [comment]
@murphyte 

Hm. Given the various issues that have relates to aspgd recently and it's movement from the old resource to GOA, I suspect that there is a connection. For example:

https://github.com/geneontology/pipeline/issues/263
https://github.com/geneontology/go-site/issues/1763
https://github.com/geneontology/go-site/issues/1820

I'm going to tag on @pgaudet, who might have more insight into what is currently available and what the future may look like.

**Comment by pgaudet** [comment]
A nidulans is now managed by GOA, which means that GO Central doesn't provide the species-specific file anymore. GOA produced a file, available at http://ftp.ebi.ac.uk/pub/databases/GO/goa/proteomes/

http://ftp.ebi.ac.uk/pub/databases/GO/goa/proteomes/22026.E_nidulans.goa

Is this what you were looking for? 

Thanks, Pascale

**Reply by murphyte** [reply]
Thanks, I think that should do it. The new file doesn't have quite the same set of synonyms in column 11. Ideally everything would include the AN### identifiers. But we'll try to work out some on-the-fly format conversion, since the numbers seem to be the same. 

Best regards,
-Terence
